frontend/www/js/omegaup/components/qualitynomination/List.vue

```diff
@@ -157,6 +185,37 @@ export default class QualityNominationList extends Vue {
     author_username: T.wordsAuthor,
   };
 
+  get orderedNominations(): types.NominationListItem[] {
+    let order = this.sortOrder === omegaup.SortOrder.Ascending ? -1 : 1;
+
+    switch (this.columnName) {
+      case 'title':
+        return this.nominations.sort((a, b) =>
+          a.problem.title < b.problem.title
+            ? order
+            : b.problem.title < a.problem.title
+            ? -1 * order
+            : 0,
+        );
+      case 'time':
+        return this.nominations.sort((a, b) =>
+          a.time.getTime() < b.time.getTime()
+            ? order
+            : b.time.getTime() < a.time.getTime()
+            ? -1 * order
+            : 0,
+        );
+      default:
```

Comment:
```suggestion
      case 'title':
      default:
```

así seguimos siendo explícitos y se de-duplica el código.

Reply:
Totalmente de acuerdo, gracias por la observación.